doc/api/events.md

```diff
@@ -913,6 +913,7 @@ const { on, EventEmitter } = require('events');
     // if concurrent execution is required.
     console.log(event); // prints ['bar'] [42]
   }
+  throw new Error('Unreachable');
```

Comment:
Perhaps a comment would suffice as I'm not sure most people would need to add this to their code if they're modelling their code based on this example?

Reply:
Or maybe the comment inside the loop could be expanded to be more clear that it loops forever?

Reply:
the comment also ok, and this line just for remind users.🤔

Reply:
I think a comment is a good idea.